redbot/cogs/mod/kickban.py

```diff
@@ -205,15 +205,18 @@ class KickBanMixin(MixinMeta):
         self,
         ctx: commands.Context,
         user: discord.Member,
-        days: Optional[int] = 0,
+        days: Optional[int] = None,
         *,
         reason: str = None,
     ):
         """Ban a user from this server and optionally delete days of messages.
 
         If days is not a number, it's treated as the first word of the reason.
-        Minimum 0 days, maximum 7. Defaults to 0."""
-
+        Minimum 0 days, maximum 7. If not specified, and defaultdays is set,
+        defaultdays will be used instead."""
```

Comment:
```suggestion
        it will be used instead."""
```
Let's not repeat the same thing twice.

Reply:
I would change the whole "If not specified, and defaultdays is set, defaultdays will be used instead." to "If not specified, defaultdays will be used instead.", because it is always set to something.